src/scss/sections/_product-carousel.scss

```diff
@@ -0,0 +1,138 @@
+.product-carousel {
+  .shop-all {
+    min-width: calc-em(154);
+    @include media-breakpoint-up(lg) {
+      min-width: calc-em(216);
+    }
+  }
+
+  .carousel--centered {
+    padding-top: map-get($spacers, 2);
+  }
+}
+
+.product-card {
+  min-height: calc-em(533);
+
+  &__title {
+    font-size: calc-em(22);
+    padding: 0;
+    margin-top: auto;
+    min-height: calc-em(54);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__price {
+    margin-top: auto;
+
+    .price {
+      font-size: calc-em(22);
+    }
+  }
+
+  &__badge {
+    position: absolute;
+    width: calc-em(66);
+    height: calc-em(66);
```

Comment:
these should be declared as badge element and maybe you can use bootstrap variables